src/pages/instances/forms/RootStorageForm.tsx

```diff
@@ -83,6 +84,7 @@ const RootStorageForm: FC<Props> = ({ formik, project }) => {
       return {
         label: storagePool.name,
         value: storagePool.name,
+        disabled: false,
```

Comment:
Isn't this false by default? Do we need to specify it?

Reply:
It's a way to keep the types similar without specifying the type of the objects in the options array.